Formula/k/kin.rb

```diff
@@ -0,0 +1,44 @@
+class Kin < Formula
+  include Language::Python::Virtualenv
+
+  desc "Sane PBXProj files"
+  homepage "https://github.com/Serchinastico/Kin"
+  url "https://files.pythonhosted.org/packages/fe/36/f83076675f5542af2418d3c75b1f4b5de9b895c37e418a086ce40a05f1bb/kin-2.1.5.tar.gz"
+  sha256 "6ee259cefa1ef7e565562f226c4fe4647990fa957b2c172b633afad3fd41c89d"
+  license "Apache-2.0"
+  head "https://github.com/Serchinastico/Kin.git", branch: "main"
+
+  depends_on "python-setuptools"
```

Comment:
this should be build-only:
```suggestion
  depends_on "python-setuptools" => :build
```

Reply:
I believe this initially failed.